### DISCUSSION TITLE
Downloaded media are deleted when I download another week

### DISCUSSION THREAD

**Discussion by Unviray** [sircharlo/meeting-media-manager#134] [Q&A] [answered]
I want to download all media in 4 next week but when I download another week all media already downloaded get deleted

**Comment by sircharlo** [comment, marked as answer]
Hi there! Just to understand your situation better, are you using congregation-level syncing?

**Reply by sircharlo** [reply]
Implemented a sort of fix for this in https://github.com/sircharlo/jw-meeting-media-fetcher/commit/3bea859ed902dc02b3e867b371aa617b5733649e; released in #141
